debugclient/main.c

```diff
@@ -178,55 +181,78 @@ int main(int argc, char *argv[])
 		switch (opt) {
 			case 'h':
 				printf("\nUsage:\n");
-				printf("\tdebugclient [-h] [-p port] [-v]\n");
+				printf("\tdebugclient [-h] [-p port] [-v] [-1] [-4] [-6]\n");
 				printf("\t-h\tShow this help\n");
 				printf("\t-p\tSpecify the port to listen on (default = 9000)\n");
 				printf("\t-v\tShow version number and exit\n");
 				printf("\t-1\tDebug once and then exit\n");
+				printf("\t-4\tListen on IPv4 (default)\n");
+				printf("\t-6\tListen on IPv6\n");
+				printf("\n");
 				exit(0);
 				break;
 			case 'v':
 				exit(0);
 				break;
 			case 'p':
 				port = atoi(optarg);
-				printf("Listening on TCP port %d.\n", port);
 				break;
 			case '1':
 				debug_once = 0;
 				break;
+			case '4':
+				ipversion = 1;
```

Comment:
I'd rather see constants for this - and, in any case, wouldn't it make sense to use 4 and 6 for its values, instead of 1 and 2?